db/migrate/20170414132105_add_language_to_statuses.rb

```diff
@@ -0,0 +1,5 @@
+class AddLanguageToStatuses < ActiveRecord::Migration[5.0]
+  def change
+    add_column :statuses, :language, :string, null: false, default: 'en'
```

Comment:
Is that true for a server running with a different default locale?

Like, if an admin has bothered to set a different default locale value, it might be a safe assumption that the statuses created on that instance are in that locale, and that this migration should use that value instead of hardcoding `en`?

When this is deployed it will be run on each instance separately, so it might be "more correct" to make the assumption based on that (ie, are the instances running in Japan, Hong Kong, etc - primarily using `en` or other locale?)